src/main/java/stsjorbsmod/cards/cull/Discordance.java

```diff
@@ -0,0 +1,68 @@
+package stsjorbsmod.cards.cull;
```

Comment:
It would be nice to give this an mspaint beta art that is distinct from `Drain Life`'s; the art quality doesn't matter at all (a stick figure drawing is fine), but it's helpful for the player to be able to distinguish cards at a glance

Reply:
Created beta art for "Discord Dance"